sqtom/fitting_1d.py

```diff
@@ -25,23 +25,90 @@
 """
 
 
+import warnings
 import numpy as np
+from scipy.optimize import root_scalar
 from lmfit import Minimizer, Parameters
 from sqtom.forward_solver import degenerate_pmf
 
 
+def two_schmidt_mode_guess(pd_data, sq_label="sq_", noise_fraction=0.001, atol=1e-8):
+    """Given a single mode histogram, this function generates a "physically" motivated guess for the loss, Schmidt
+    occupations and dark counts parameters.
+
+    This model is sensible only if there is a probability for an n == 2 event. If there are only probabilities for
+    n == 0 and n == 1 then the g2 will erroneously be calculated as 0 and as a result the remainder of the estimtes will
```

Comment:
(Minor suggestion): might be worth describing what is meant by `an n == 2 event`, etc. (just to make it more explicitly explained).

Reply:
I have changed the docstring to remove this possibly  confusing wording.